var/spack/repos/builtin/packages/rocm-debug-agent/package.py

```diff
@@ -15,23 +15,33 @@ class RocmDebugAgent(CMakePackage):
 
```

Comment:
you can remove this line

Reply:
Is it this line?. This line is to use cmake template
class RocmDebugAgent(CMakePackage):

am I missing something here?

Reply:
url      = "https://github.com/ROCm-Developer-Tools/rocr_debug_agent/archive/roc-3.5.0.tar.gz"